### ISSUE TITLE
Mario cant interact with some smaller objects.

### ISSUE THREAD

**Issue by RjayTheDoggo** [Retro64Mod/Retro64Mod.github.io#4]
For example things like stacked snow, pots and end rods. I totally did not try to replicate stairs form the game so i can blj haha.

**Comment by dylanpdx** [comment]
Got it. Will take a look!

**Comment by Roshibomb** [comment]
I decided to test a bunch of items from the Creative inventory, and currently, these ones either do not have any collision or have jank collision in certain scenarios (mostly the former). Head and offhand slots included (I ran out of space). All of the items that have multiple variants probably affect all variants (different wood types and such). This isn't a conclusive list, just whatever I thought would probably have weird collision.
![image](https://user-images.githubusercontent.com/58871286/159812685-3005a18f-9583-4674-9cd6-b1a5b097724b.png)
